package.json

```diff
@@ -30,22 +30,29 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
-    "@nrwl/angular": "12.9.0",
+    "@angular-devkit/architect": "0.1201.2",
+    "@angular-devkit/build-angular": "12.1.2",
+    "@angular-devkit/core": "12.1.2",
+    "@nrwl/angular": "^12.9.0",
     "@nrwl/cli": "12.9.0",
     "@nrwl/eslint-plugin-nx": "12.9.0",
     "@nrwl/jest": "12.9.0",
     "@nrwl/node": "12.9.0",
     "@nrwl/tao": "12.9.0",
     "@nrwl/workspace": "12.9.0",
     "@types/jest": "27.0.2",
+    "@types/lodash": "^4.14.174",
     "@types/node": "^14.17.19",
-    "@typescript-eslint/eslint-plugin": "4.32.0",
-    "@typescript-eslint/parser": "4.32.0",
+    "@typescript-eslint/eslint-plugin": "4.28.4",
+    "@typescript-eslint/parser": "4.28.4",
     "dotenv": "10.0.0",
     "eslint": "7.32.0",
     "eslint-config-prettier": "8.3.0",
+    "gherkin-testcafe": "5.1.0",
     "jest": "27.2.2",
+    "lodash": "^4.17.21",
```

Comment:
can it be change to https://www.npmjs.com/package/lodash.ismatch? as I use only that function maybe it is not necessary to drag whole lodash :)

Reply:
nah - those are dev depndencies - thanks a lot for great work :)